### DISCUSSION TITLE
New notebook not showing up on locally-generated docs site

### DISCUSSION THREAD

**Discussion by alronlam** [thinkingmachines/geowrangler#161] [Q&A] [answered]
1. I created a new notebook in `notebooks/overview.poverty_mapping_demo.ipynb`
2. I followed the instructions for generating a local version of the site:
```
poetry run nbdev_build_docs --mk_readme False --force_all True
docker-compose up jekyll
```
3. Visiting `localhost:4000` though, my new notebook didn't show up. Am I missing a step?
<img width="953" alt="missing_overview" src="https://user-images.githubusercontent.com/1049495/188068957-49770541-efea-47b2-84a7-b6a1e4309907.PNG">

**Comment by jtmiclat** [comment]
The document might be rendered but the sidebar is for sure missing it. Please add the links to the sidebar config found in `docs/sidebar.json`

**Comment by butchtm** [comment, marked as answer]
@alronlam when you run the `nbdev_build_docs` command, there should be a `.html` file generated for each notebook in the `docs` directory -- you can just access the new one directly via localhost. Then add this generated html file in the `sidebar.json` file.
